packages/cloudbuster/src/findings.ts

```diff
@@ -103,6 +103,7 @@ export function groupFindingsByAccount(findings: Finding[]): GroupedFindings {
 		if (!findingsGroupedByAwsAccount[awsAccountId]) {
 			findingsGroupedByAwsAccount[awsAccountId] = [];
 		}
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- to allow ts version bump
 		findingsGroupedByAwsAccount[awsAccountId]?.push(finding);
```

Comment:
```suggestion
		findingsGroupedByAwsAccount[awsAccountId].push(finding);
```
The previous `if` statement already checks for existence, so the linter is warning that the `?` is unnecessary. Let's just remove it :)